docs/index.rst

```diff
@@ -11,7 +11,14 @@ Welcome to Foreman Ansible Modules's documentation!
    :caption: User documentation
 
    README
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Contained plugins
```

Comment:
I'm wondering whether this should have a caption or not.

![Screenshot from 2020-07-20 13-16-52](https://user-images.githubusercontent.com/94284/87932222-70cd7780-ca8b-11ea-802e-3870b7d24569.png)
![Screenshot from 2020-07-20 13-17-03](https://user-images.githubusercontent.com/94284/87932229-732fd180-ca8b-11ea-9329-a5338a78b12f.png)

It has to be an own `toctree` so that we can set `maxdepth: 1`, as otherwise Sphinx will add the subsections of the callback and inventory plugins, which are not useful in this TOC, while having `maxdepth: 2` for the `README` above *is* useful.